pennylane/qnodes/base.py

```diff
@@ -689,6 +674,7 @@ def _default_args(self, kwargs):
 
             # The following is a check of the default parameter which works for numpy
             # arrays as well (if it is a numpy array, each element is checked separately).
+            # FIXME why are numpy array default values not good automatically?
```

Comment:
This was a bugfix made by @antalszava, he might have more details as to why the following was required

Reply:
This change was made in https://github.com/XanaduAI/pennylane/pull/440, when the old QNodes were removed and the new QNodes put in their place. Off the top of my head, some of the original QNode tests testing argument handling were failing after this change, but I cannot recall which one.